cluster/images/hyperkube/BUILD

```diff
@@ -7,21 +7,24 @@ container_image(
         "//cmd/hyperkube",
     ],
     stamp = True,
+    # These exist in the base image, but we may as well be explicit here too.
```

Comment:
Should we actually? 🤷‍♂️

Reply:
Seems like this adds _some_ build time cost and possible inconsistency for no major gain

Reply:
I was making a point at how bazel doesn't need qemu to do this, though also bazel only supports linux/amd64 for now, sooooo

Reply:
(removed)